lib/resty/auto-ssl/ssl_certificate.lua

```diff
@@ -113,9 +113,11 @@ local function get_cert(auto_ssl_instance, domain)
   end
 
   -- Finally, issue a new certificate if one hasn't been found yet.
-  fullchain_pem, privkey_pem = issue_cert(auto_ssl_instance, storage, domain)
-  if fullchain_pem and privkey_pem then
-    return convert_to_der_and_cache(domain, fullchain_pem, privkey_pem, true)
+  if ssl_options and ssl_options["generate_certs"] ~= false then
+    fullchain_pem, privkey_pem = issue_cert(auto_ssl_instance, storage, domain)
+    if fullchain_pem and privkey_pem then
+      return convert_to_der_and_cache(domain, fullchain_pem, privkey_pem, true)
+    end
   end
```

Comment:
`get_cert` returns an error message as the very last parameter. Please add a appropriate message telling the users what is going on, in case certificates are not being generated because of this setting:

```
return nil, nil, nil, "did not issue certificate, because the generate_certs setting is false"
```